### PR TITLE
created function to allow change default values, fix loading search users

### PR DESCRIPTION
**Pull request by ggazzo**

### CODE REVIEW

File: app/lib/client/lib/formatDate.js [hunk 1]
```diff
@@ -1,21 +1,32 @@
...
+let sameDay;
+const dayFormat = ['h:mm A', 'H:mm'];
+
+Tracker.autorun(() => {
+	clockMode = getUserPreference(Meteor.userId(), 'clockMode', false);
+	sameDay = dayFormat[clockMode - 1] || 'h:mm A';
+	lastDay = t('yesterday');
+});
+
 export const formatTime = (time) => {
```
Comment (tassoevan) [inline app/lib/client/lib/formatDate.js]: Is this helper still used somewhere?

File: app/ui/client/views/app/room.js [hunk 2]
```diff
@@ -697,7 +697,7 @@ Template.room.events({
 		const hasMoreNext = RoomHistoryManager.hasMoreNext(this._id);
 
 		if ((isLoading === false && hasMore === true) || hasMoreNext === true) {
-			if (hasMore === true && lastScrollTop <= height / 3) {
+			if (hasMore === true && lastScrollTop <= height / 4) {
```
Comment (tassoevan) [inline app/ui/client/views/app/room.js]: Assign the `height / 4` expression to a variable to make it more explicit as a scrolling threshold.

File: package-lock.json [hunk 3]
```diff
@@ -5542,7 +5542,7 @@
 		},
 		"doctrine": {
 			"version": "1.5.0",
-			"resolved": "https://registry.npmjs.org/doctrine/-/doctrine-1.5.0.tgz",
+			"resolved": "http://registry.npmjs.org/doctrine/-/doctrine-1.5.0.tgz",
```
Comment (tassoevan) [inline package-lock.json]: Check if npm resolved these URLs from `https` to `http` before commit `package-lock.json`.

File: app/lib/client/lib/formatDate.js [hunk 4]
```diff
@@ -1,21 +1,32 @@
...
+let lastDay = t('yesterday');
+let clockMode;
+let sameDay;
+const dayFormat = ['h:mm A', 'H:mm'];
+
+Tracker.autorun(() => {
+	clockMode = getUserPreference(Meteor.userId(), 'clockMode', false);
+	sameDay = dayFormat[clockMode - 1] || 'h:mm A';
+	lastDay = t('yesterday');
+});
```
[the remark this answers is not in the document]
Comment (ggazzo) [reply app/lib/client/lib/formatDate.js]: not necessarily

**Pull request merged**